docs/source/essentials/server.md

```diff
@@ -159,11 +95,55 @@ And the server should return a simple response:
 ```json
 {
   "data": {
-    "hello": null
+    "hello": "world"
   }
 }
 ```
 
+Your server itself is hosted at http://localhost:4000/graphql. This would be the endpoint you pass to set up Apollo Client.
+
+<h2 id="integrations">Server integrations</h2>
+
+Depending on whether we are creating a new application or an existing application, the method of importing `apollo-server` will vary slightly since Apollo Server must adapt to the semantics of existing servers (e.g. Express, Hapi, etc.)
+
+Both import methods will use the `apollo-server` module we installed in the previous step, but existing applications will append the desired middleware as a path-based import (e.g. `apollo-server/<variant>`, where `<variant>` would be `express`, `koa`, etc.).
+
+<h3 id="middleware">Middleware</h3>
+
+Existing applications generally already have existing middleware in place and Apollo Server works with those existing middleware options.  But in order to play along, we need to import the `ApolloServer` which corresponds to the type of server which is already in use.
```

Comment:
>   But in order to play along,

Can we rephrase this part of the sentence?

Reply:
good catch 👍